static/node/package.json

```diff
@@ -1,19 +1,18 @@
 {
-    "dependencies": {
-        "@ganache/filecoin": "0.1.5",
-        "@truffle/config": "1.2.17",
-        "@truffle/decoder": "3.0.16",
-        "electron": "12.0.2",
-        "electron-fetch": "1.7.3",
-        "fs-extra": "9.0.0",
-        "ganache": "7.0.0-filecoin.6",
-        "ganache-core": "2.13.1",
-        "lodash.clonedeep": "4.5.0",
-        "lodash.merge": "4.6.2",
-        "lodash.padstart": "4.6.1",
-        "temp": "0.9.1",
-        "web3-eth-abi": "1.2.6",
-        "web3-providers-http": "1.2.6",
-        "web3-providers-ws": "1.2.6"
-    }
+  "dependencies": {
+    "@ganache/filecoin": "0.1.5",
+    "@truffle/config": "1.2.17",
+    "@truffle/decoder": "3.0.16",
+    "electron": "12.0.2",
+    "electron-fetch": "1.7.3",
+    "fs-extra": "9.0.0",
+    "ganache": "^7.5.0",
```

Comment:
Should `ganache-core` be in this package.json still?

Reply:
Yes! Should probs also pin the "ganache" dependency.